tests/integration/orders-update.spec.js

```diff
@@ -40,6 +43,7 @@ test('the module should modify an existing order', (t) => {
       ),
     )
     // Create needed data
+    .then(() => ordersUpdate.client.channels.ensure(channelKey, channelRole))
```

Comment:
So we don't need to delete it anymore?
If it's nice we should use it for the others too.